tailwind.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
   darkMode: false,
   theme: {
     extend: {
+      spacing: {
+        financemargin: '24rem',
```

Comment:
do not add page specific styling to tailwind config. Instead add the appropriate tailwind class to the Finance component